src/core/server/metrics/collectors/os.ts

```diff
@@ -43,20 +55,30 @@ export class OsMetricsCollector implements MetricsCollector<OpsOsMetrics> {
         used_in_bytes: os.totalmem() - os.freemem(),
       },
       uptime_in_millis: os.uptime() * 1000,
+      ...(await this.getDistroStats(platform)),
+      ...(await this.cgroupCollector.collect()),
```

Comment:
Should we add a test in `os.test.ts` to ensure the collector is properly called, or is this unnecessary?